dts/riscv/ite/it8801-mfd-gpiocr.dtsi

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	soc {
+		gpio0: gpiocr@0a {
```

Comment:
Nodename `gpio0` is likely to conflict the the GPIO controller on the SoC.

Reply:
Renamed.